lib/timescaledb/database/schema_statements.rb

```diff
@@ -133,6 +133,65 @@ def remove_reorder_policy_sql(hypertable, **options)
         "SELECT remove_reorder_policy(#{arguments.join(', ')});"
       end
 
+      # @see https://docs.timescale.com/api/latest/continuous-aggregates/create_materialized_view
+      #
+      # @param [String] continuous_aggregate The name of the continuous aggregate view to be created
+      # @param [Hash] options The optional arguments
+      # @return [String] The create materialized view SQL statement
+      def create_continuous_aggregate_sql(continuous_aggregate, sql, **options)
+        options.transform_keys!(&:to_sym)
+
+        with_data_opts = %w[WITH DATA]
+        with_data_opts.insert(1, 'NO') if options.key?(:with_no_data)
+
+        <<~SQL
+          CREATE MATERIALIZED VIEW #{continuous_aggregate}
+          WITH (timescaledb.continuous) AS
+          #{sql.strip}
+          #{with_data_opts.join(' ')};
+        SQL
+      end
+
+      # @see https://docs.timescale.com/api/latest/continuous-aggregates/drop_materialized_view
+      #
+      # @param [String] continuous_aggregate The name of the continuous aggregate view to be dropped
+      # @return [String] The drop materialized view SQL statement
+      def drop_continuous_aggregate_sql(continuous_aggregate)
+        "DROP MATERIALIZED VIEW #{continuous_aggregate};"
+      end
+
+      # @see https://docs.timescale.com/api/latest/continuous-aggregates/add_continuous_aggregate_policy
+      #
+      # @param [String] continuous_aggregate The name of the continuous aggregate to add the policy for
+      # @param [Hash] options The optional arguments
+      # @return [String] The add_continuous_aggregate_policy SQL statement
+      def add_continuous_aggregate_policy_sql(continuous_aggregate, **options)
+        options.transform_keys!(&:to_sym)
+
+        unless %i[start_offset end_offset schedule_interval].all?(&options.method(:key?))
+          raise ArgumentError, 'Missing required options (start_offset, end_offset and schedule_interval)'
```

Comment:
Very good! an option would be also show what params are missing or allow the end user to configure global params for such cases.

Maybe keyword arguments would force the user to pass the right parameters. I like the idea of having flexible arguments but I don't see new arguments coming for this function.

Reply:
I was not sure about including start_offset, end_offset and schedule_interval as options here, they are required in the documentation, I got confused because you can pass nil to start_offset and end_offset, saying that, I like the idea of using keyword arguments but I think I would only use that for required params, what do you think? 

And I'd change this signature to be 👇 

```ruby
      def add_continuous_aggregate_policy_sql(continuous_aggregate, start_offset: nil, end_offset: nil, schedule_interval:, **options)
```